.pre-commit-config.yaml

```diff
@@ -21,3 +21,12 @@ repos:
         entry: python -m CheckLicensesInFiles
         language: python
         types: [python]
+-   repo: local
+    hooks:
+    -   id: MyPy Checks
+        name: MyPy Checks
+        description: Checks that all files have correct type annotations
+        entry: p  mypy --ignore-missing-imports --no-site-packages mantidimaging
```

Comment:
I think this should be
`entry: mypy --ignore-missing-imports --no-site-packages`

The `p` is giving me command not found

And the path is not needed as the changed files get passed in by pre-commit. (otherwise I get `error: Duplicate module` when I have a changed file).

Reply:
Should be fixed now, I hadn't realised I introduced a typo just before committing. 😅  Should by python instead of p now. I've also updated the path.  p worked for me as I have it aliased to python as i don't trust my ability to type python all the time